woocommerce/wc-admin
# Remove no-reply from inbox notification emails

Fixes https://github.com/woocommerce/woocommerce-admin/issues/6362

Overrides [`get_headers`](https://github.com/woocommerce/woocommerce/blob/9398c894c943524928d5b181eaeba9b44eb7a8ad/includes/emails/class-wc-email.php#L410) function from WC_Email class to remove `reply-to` header.

### Detailed test instructions:

The following tests using AddFirstProduct email note notification:

- Create a zip for testing with `npm run test:zip`.
- Create a `jurassic.ninja` instance.
- Upload the plugin and activate it.
- Update the installation date (we need a store between 2 and 5 days old). You can do it with an SQL statement like this:

```
UPDATE `wp_options` SET `option_value`=UNIX_TIMESTAMP(DATE_SUB(NOW(), INTERVAL 4 day)) WHERE `option_name` = 'woocommerce_admin_install_timestamp';
```

- Make sure the `woocommerce_merchant_email_notifications` option is set to `yes`:
```
UPDATE `wp_options` SET `option_value` = 'yes' WHERE `wp_options`.`option_name` = 'woocommerce_merchant_email_notifications';
```

- Run the `wc_admin_daily ` cron job (this tool can help [WP Crontrol](https://wordpress.org/plugins/wp-crontrol/)).
- You should receive an email, check the header and make sure `no-reply` is not present.

### Screenshot

<img src="https://user-images.githubusercontent.com/3747241/112135490-cd7bc600-8c08-11eb-8be4-42ad29a62a48.png" width="600">
